java/src/main/java/com/xebia/functional/xef/java/auto/AIScope.java

```diff
@@ -32,6 +36,7 @@ public class AIScope implements AutoCloseable {
     private final KtorOpenAIClient client;
     private final Embeddings embeddings;
     private final VectorStore vectorStore;
```

Comment:
Fields:
private final Embeddings embeddings;
private final VectorStore vectorStore;

they can be converted to a local variables, you're just using them in the constructor:
AIScope(ObjectMapper om, OpenAIConfig config)